tasks/main.yml

```diff
@@ -46,96 +82,139 @@
     state: directory
     mode: "0755"
 
-- name: Generate a configuration for kernel settings
-  template:
-    src: "{{ __kernel_settings_profile_src }}.j2"
-    dest: "{{ __kernel_settings_profile_filename }}"
-    force: false
-    mode: "0644"
+- name: Get active_profile
+  slurp:
+    path: "{{ __kernel_settings_tuned_active_profile }}"
+  register: __kernel_settings_tuned_current_profile
+
+- name: Set active_profile
+  set_fact:
+    __kernel_settings_active_profile: "{{ __active_profile }}"
```

Comment:
You can use what is set for `__active_profile` here directly

Reply:
not sure what you mean

Reply:
I mean do not set `__active_profile` in vars, just paste what `__active_profile` equals right in `__kernel_settings_active_profile`.